CRITICAL - 2023-10-30 09:11:57 --> Undefined variable: count_pembimbing
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(46): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'D:\\xampp\\htdocs...', 46, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 09:11:58 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 09:14:35 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-10-30 09:38:46 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(308): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:40:33 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(308): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:40:34 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(308): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2023-10-30 09:41:46 --> ErrorException: pg_query(): Query failed: ERROR:  table name &quot;sspelayanan&quot; specified more than once in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT *\nFROM "...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 D:\xampp\htdocs\peluit\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 D:\xampp\htdocs\peluit\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(303): CodeIgniter\BaseModel->findAll()
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#9 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-30 09:41:46 --> pg_query(): Query failed: ERROR:  table name &quot;sspelayanan&quot; specified more than once
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT *\nFROM "...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 D:\xampp\htdocs\peluit\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 D:\xampp\htdocs\peluit\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(303): CodeIgniter\BaseModel->findAll()
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#9 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2023-10-30 09:41:47 --> ErrorException: pg_query(): Query failed: ERROR:  table name &quot;sspelayanan&quot; specified more than once in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT *\nFROM "...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 D:\xampp\htdocs\peluit\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 D:\xampp\htdocs\peluit\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(303): CodeIgniter\BaseModel->findAll()
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#9 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-30 09:41:47 --> pg_query(): Query failed: ERROR:  table name &quot;sspelayanan&quot; specified more than once
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT *\nFROM "...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 D:\xampp\htdocs\peluit\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 D:\xampp\htdocs\peluit\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(303): CodeIgniter\BaseModel->findAll()
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#9 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2023-10-30 09:43:20 --> ErrorException: pg_query(): Query failed: ERROR:  table name &quot;sspelayanan&quot; specified more than once in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #131, 'SELECT *\nFROM "...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 D:\xampp\htdocs\peluit\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 D:\xampp\htdocs\peluit\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(303): CodeIgniter\BaseModel->findAll()
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#9 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-30 09:43:20 --> pg_query(): Query failed: ERROR:  table name &quot;sspelayanan&quot; specified more than once
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #131, 'SELECT *\nFROM "...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 D:\xampp\htdocs\peluit\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 D:\xampp\htdocs\peluit\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(303): CodeIgniter\BaseModel->findAll()
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#9 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2023-10-30 09:43:21 --> ErrorException: pg_query(): Query failed: ERROR:  table name &quot;sspelayanan&quot; specified more than once in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #131, 'SELECT *\nFROM "...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 D:\xampp\htdocs\peluit\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 D:\xampp\htdocs\peluit\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(303): CodeIgniter\BaseModel->findAll()
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#9 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-30 09:43:21 --> pg_query(): Query failed: ERROR:  table name &quot;sspelayanan&quot; specified more than once
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #131, 'SELECT *\nFROM "...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 D:\xampp\htdocs\peluit\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 D:\xampp\htdocs\peluit\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(303): CodeIgniter\BaseModel->findAll()
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#9 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2023-10-30 09:46:05 --> ErrorException: pg_query(): Query failed: ERROR:  table name &quot;sspelayanan&quot; specified more than once in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT *\nFROM "...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 D:\xampp\htdocs\peluit\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 D:\xampp\htdocs\peluit\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(305): CodeIgniter\BaseModel->findAll()
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#9 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-30 09:46:05 --> pg_query(): Query failed: ERROR:  table name &quot;sspelayanan&quot; specified more than once
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT *\nFROM "...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 D:\xampp\htdocs\peluit\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 D:\xampp\htdocs\peluit\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(305): CodeIgniter\BaseModel->findAll()
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#9 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2023-10-30 09:46:06 --> ErrorException: pg_query(): Query failed: ERROR:  table name &quot;sspelayanan&quot; specified more than once in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT *\nFROM "...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 D:\xampp\htdocs\peluit\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 D:\xampp\htdocs\peluit\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(305): CodeIgniter\BaseModel->findAll()
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#9 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-30 09:46:06 --> pg_query(): Query failed: ERROR:  table name &quot;sspelayanan&quot; specified more than once
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT *\nFROM "...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 D:\xampp\htdocs\peluit\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 D:\xampp\htdocs\peluit\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(305): CodeIgniter\BaseModel->findAll()
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#9 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2023-10-30 09:46:07 --> ErrorException: pg_query(): Query failed: ERROR:  table name &quot;sspelayanan&quot; specified more than once in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT *\nFROM "...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 D:\xampp\htdocs\peluit\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 D:\xampp\htdocs\peluit\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(305): CodeIgniter\BaseModel->findAll()
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#9 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-30 09:46:07 --> pg_query(): Query failed: ERROR:  table name &quot;sspelayanan&quot; specified more than once
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT *\nFROM "...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 D:\xampp\htdocs\peluit\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 D:\xampp\htdocs\peluit\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(305): CodeIgniter\BaseModel->findAll()
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#9 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#10 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-30 09:46:41 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:46:42 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:46:44 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:46:50 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:46:50 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:47:46 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:47:46 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:47:47 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:47:47 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:47:47 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:47:48 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:47:48 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:47:48 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:47:49 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:47:49 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:47:50 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:47:52 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:48:33 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:48:34 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:48:35 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:48:35 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:48:36 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:48:36 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:48:37 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:48:37 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:48:37 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:48:38 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:48:38 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 09:48:39 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 10:06:57 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 10:06:57 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\View\View.php(214): include()
#1 D:\xampp\htdocs\peluit\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 D:\xampp\htdocs\peluit\system\Common.php(1097): CodeIgniter\View\View->render('admin/landing', Array, true)
#3 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(310): view('admin/landing', Array)
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#5 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-10-30 10:09:07 --> Array to string conversion
#0 D:\xampp\htdocs\peluit\app\Controllers\Admin.php(301): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'D:\\xampp\\htdocs...', 301, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 10:09:20 --> Object of class stdClass could not be converted to string
#0 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-10-30 10:09:21 --> Object of class stdClass could not be converted to string
#0 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-10-30 10:09:39 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-10-30 10:09:39 --> Cannot use object of type stdClass as array
#0 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Admin->landing_page()
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
ERROR - 2023-10-30 10:49:27 --> ErrorException: pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^ in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(107): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:49:27 --> pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(107): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:49:27 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 10:49:27 --> count(): Parameter must be an array or an object that implements Countable
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'count(): Parame...', 'D:\\xampp\\htdocs...', 62, Array)
#1 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(62): count(Object(stdClass))
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2023-10-30 10:49:27 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 10:49:28 --> count(): Parameter must be an array or an object that implements Countable
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'count(): Parame...', 'D:\\xampp\\htdocs...', 62, Array)
#1 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(62): count(Object(stdClass))
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2023-10-30 10:49:28 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
ERROR - 2023-10-30 10:49:28 --> ErrorException: pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^ in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(107): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:49:28 --> pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(107): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:49:28 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 10:49:35 --> count(): Parameter must be an array or an object that implements Countable
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'count(): Parame...', 'D:\\xampp\\htdocs...', 62, Array)
#1 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(62): count(Object(stdClass))
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#4 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2023-10-30 10:50:28 --> Undefined property: stdClass::$akronim
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(62): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'D:\\xampp\\htdocs...', 62, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 10:50:29 --> Undefined property: stdClass::$akronim
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(62): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'D:\\xampp\\htdocs...', 62, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
ERROR - 2023-10-30 10:51:15 --> ErrorException: pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^ in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(107): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:51:15 --> pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(107): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:51:15 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
ERROR - 2023-10-30 10:51:22 --> ErrorException: pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^ in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(107): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:51:22 --> pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(107): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:51:22 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
ERROR - 2023-10-30 10:51:44 --> ErrorException: pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^ in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(107): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:51:44 --> pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(107): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:51:44 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
ERROR - 2023-10-30 10:51:46 --> ErrorException: pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^ in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(107): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:51:46 --> pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(107): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:51:46 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
ERROR - 2023-10-30 10:52:13 --> ErrorException: pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^ in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(107): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:52:13 --> pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(107): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:52:13 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
ERROR - 2023-10-30 10:57:27 --> ErrorException: pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^ in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(108): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:57:27 --> pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(108): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:57:27 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
ERROR - 2023-10-30 10:57:30 --> ErrorException: pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^ in D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(108): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:57:30 --> pg_query(): Query failed: ERROR:  syntax error at or near &quot;)&quot;
LINE 3: WHERE &quot;tb_tiket&quot;.&quot;id_pelayanan&quot; IN ()
                                            ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'D:\\xampp\\htdocs...', 136, Array)
#1 D:\xampp\htdocs\peluit\system\Database\Postgre\Connection.php(136): pg_query(Resource id #125, 'SELECT COUNT(*)...')
#2 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT COUNT(*)...')
#3 D:\xampp\htdocs\peluit\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#4 D:\xampp\htdocs\peluit\system\Database\BaseBuilder.php(1527): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#5 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(108): CodeIgniter\Database\BaseBuilder->countAllResults()
#6 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_tiket()
#7 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#8 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-10-30 10:57:30 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 11:38:14 --> Trying to get property 'id_sub' of non-object
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(60): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 60, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 11:38:14 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 11:38:27 --> Trying to get property 'id_sub' of non-object
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(60): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 60, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 11:38:27 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 11:40:44 --> Trying to get property 'id_sub' of non-object
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(60): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 60, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 11:40:44 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 11:41:43 --> Trying to get property 'id_sub' of non-object
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(60): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 60, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 11:41:43 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 11:41:45 --> Trying to get property 'id_sub' of non-object
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(60): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 60, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 11:41:45 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 11:41:56 --> Trying to get property 'id_sub' of non-object
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(60): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 60, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 11:41:56 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 11:42:21 --> Trying to get property 'id_sub' of non-object
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(60): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 60, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 11:42:21 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 11:43:16 --> Trying to get property 'id_sub' of non-object
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(60): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 60, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 11:43:16 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 11:43:24 --> Trying to get property 'id_sub' of non-object
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(60): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 60, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 11:43:24 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 12:16:34 --> Trying to get property 'id_sub' of non-object
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(60): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 60, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 12:16:35 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 12:49:20 --> Trying to get property 'id_sub' of non-object
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(60): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 60, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 12:49:20 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 13:10:36 --> Call to undefined method App\Controllers\Auth::telegram()
#0 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Auth->get_username()
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-10-30 13:10:40 --> Call to undefined method App\Controllers\Auth::telegram()
#0 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Auth->get_username()
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-10-30 13:22:11 --> Call to undefined method App\Controllers\Auth::telegram()
#0 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Auth->get_username()
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-10-30 13:22:18 --> Call to undefined method App\Controllers\Auth::telegram()
#0 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Auth->get_username()
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-10-30 13:22:53 --> Call to undefined function App\Controllers\telegram()
#0 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Auth->get_username()
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-10-30 13:25:11 --> Call to undefined function App\Controllers\telegram()
#0 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Auth->get_username()
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-10-30 13:27:36 --> Call to undefined function App\Controllers\telegram()
#0 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Auth->get_username()
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2023-10-30 13:35:11 --> Trying to get property 'id_sub' of non-object
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(60): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 60, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 13:35:11 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 13:36:39 --> Trying to get property 'id_sub' of non-object
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(60): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 60, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 13:36:39 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 13:36:40 --> Trying to get property 'id_sub' of non-object
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(60): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 60, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 13:36:40 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2023-10-30 13:36:41 --> Trying to get property 'id_sub' of non-object
#0 D:\xampp\htdocs\peluit\app\Controllers\Operator.php(60): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'D:\\xampp\\htdocs...', 60, Array)
#1 D:\xampp\htdocs\peluit\system\CodeIgniter.php(824): App\Controllers\Operator->orders_user()
#2 D:\xampp\htdocs\peluit\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Operator))
#3 D:\xampp\htdocs\peluit\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\xampp\htdocs\peluit\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2023-10-30 13:36:41 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "Type is not supported". in D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php:41
Stack trace:
#0 D:\xampp\htdocs\peluit\system\Format\JSONFormatter.php(41): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Type is not sup...')
#1 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(341): CodeIgniter\Format\JSONFormatter->format(Array)
#2 D:\xampp\htdocs\peluit\system\API\ResponseTrait.php(99): CodeIgniter\Debug\Exceptions->format(Array)
#3 D:\xampp\htdocs\peluit\system\Debug\Exceptions.php(110): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
